aiida/restapi/resources.py

```diff
@@ -125,6 +125,7 @@ def _load_and_verify(self, node_id=None):
 
         return node
 
+    @close_session
```

Comment:
Sure. Was looking for the most "upstream" way of introducing this session-decorator as well, so perhaps this is the best way. Can you confirm that perhaps @waychal?

Reply:
I have gone for a compromise, since I discovered the `Resource` attribute `method_decorators`, which I have initialized in the `BaseResource` to always include `close_session`.
This makes for "easy" testing as well of a REST API without the use of `close_session`, by simply sub-classing an existing `Resource` and setting the attribute to `None` (or equivalent).